code/src/sidebar.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { slide as Menu } from 'react-burger-menu';
+import data from './stretch-goal.json'
+
+console.log(data)
+
+export default () => {
+  return (
+    <Menu>
+    {data.playlists.items.map((item) => {
+      return <a><a className="album-link" href={item.external_urls.spotify} target="_blank" rel="noopener noreferrer">
+      {item.name}</a></a>
+    })}
```

Comment:
I think this would be more readable using implicit returns, and some newlines:

```js
    {data.playlists.items.map((item) => (
      <a className="album-link" href={item.external_urls.spotify} target="_blank" rel="noopener noreferrer">
        {item.name}
      </a>
    ))}
```

Also, why double `<a>` tags? One should be enough, I think ;)

One more thing - great work with the target blank technique with 'noopener noreferrer' - great practice! 👏